ui/src/pages/HomePage/index.js

```diff
@@ -44,6 +53,15 @@ class Home extends Component {
   }
 
   componentDidMount = () => {
+    const proceedOnce = sessionStorage.getItem('proceedOnce') === 'true';
+    this.setState({ proceedOnce });
+
+    const sessionStorageSetHandler = function () {
+      this.setState({ proceedOnce: true });
+    }.bind(this);
```

Comment:
@lucuilearningpool , why do you need `bind` here?

Reply:
👍